.github/workflows/daily.yaml

```diff
@@ -55,22 +55,23 @@ jobs:
     - name: run safety check
       run: safety check
 
-  notify:
-    runs-on: ubuntu-latest
+  slack-notify:
     needs:
+      - smoke-test-dev
       - safety-check-sdk
       - safety-check-endpoint
```

Comment:
I know this is pre-existing, but should we be requiring the safety-checks here? Seems like something to remove in a future version.